ios/FluentUI/Avatar/Avatar.swift

```diff
@@ -246,32 +270,63 @@ public struct Avatar: View, TokenizedControlView {
                                         )
                                 .contentShape(Circle()),
                              alignment: .center)
-                    .modifyIf(shouldDisplayPresence, { thisView in
+            }
+        }
+
+        @ViewBuilder
+        var avatar: some View {
+            if style != .group {
```

Comment:
Would it be possible to add a check for if we're showing activity or presence here, and then also move all the `let presenceMath`s down here?

Reply:
Moved!